.changeset/rude-parrots-know.md

```diff
@@ -0,0 +1,6 @@
+---
+'@celo/base': major
+'@celo/utils': major
+---
+
+remove all functions from contacts + displayformating, and some from currencies + phonenumbers
```

Comment:
Could you make this a little more descriptive for future reference? I'm thinking about folks that read the `CHANGELOG.md` and would like to grasp the scope of changes without opening the PR/commit.